admin/acceptance.py

```diff
@@ -159,6 +171,11 @@ def __init__(self):
             raise UsageError("Unsupored varianta: %s."
                              % (', '.join(self.variants),))
 
+    def ensure_keys(self, reactor):
```

Comment:
:question: I'm not sure what our coding standard is for docstrings which are covered by the interface's docstring. Maybe we reference them?